demo/config.yaml

```diff
@@ -107,3 +108,6 @@ lcr-modules:
         inputs:
             sample_bam: "data/{sample_id}.bam"
             sample_bai: "data/{sample_id}.bam.bai"
+        scratch_subdirectories: ["mpileup"]
```

Comment:
I wonder if this should be `scratch/mpileup`? Otherwise, I think it is creating the mpileup in running directory?

Reply:
The op.setup_module code makes a symlink in the control freec results directory and puts the real directory in the scratch path that's specified by the user, so it doesn't need to be `scratch/mpileup`. 

However I do recommend leaving this blank and placing a comment there recommending to users which directories should be scratch. Otherwise you get unexpected scratch directory usage and it creates many, many issues with existing symlink errors if the user needs to re-run.

Reply:
In my test demo run, it was generating the scratch folder inside the scratch space, so it seemed like it was working

Reply:
Even if this "works", as a general rule, I think we are leaving this empty (i.e. []) in default configs along with a comment to explain which rules might need scratch space. i.e. what Laura said.